operator v0.50.0 enalbe tekton-dashboard fail.

# Expected Behavior

# Actual Behavior

# Steps to Reproduce the Problem

1.
2.
3.

# Additional Info

- Kubernetes version: v1.21.1

  **Output of `kubectl version`:**

```
Client Version: version.Info{Major:"1", Minor:"22", GitVersion:"v1.22.2", GitCommit:"8b5a19147530eaac9476b0ab82980b4088bbc1b2", GitTreeState:"clean", BuildDate:"2021-09-15T21:31:32Z", GoVersion:"go1.16.8", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"21", GitVersion:"v1.21.1", GitCommit:"5e58841cce77d4bc13713ad2b91fa0d961e69192", GitTreeState:"clean", BuildDate:"2021-05-21T23:01:33Z", GoVersion:"go1.16.4", Compiler:"gc", Platform:"linux/amd64"}
```

- Tekton Pipeline version: v0.28.2

  **Output of `tkn version` or `kubectl get pods -n tekton-pipelines -l app=tekton-pipelines-controller -o=jsonpath='{.items[0].metadata.labels.version}'`**

```
Client version: 0.21.0
Pipeline version: v0.28.2
Triggers version: v0.16.0
```

- Tekton Operator version: v0.50.0


# Error logs

```
panic: open /var/run/ko/tekton-dashboard/tekton-dashboard-fullaccess: no such file or directory

goroutine 932 [running]:
github.com/tektoncd/operator/pkg/reconciler/common.latestRelease(...)
	github.com/tektoncd/operator/pkg/reconciler/common/releases.go:170
github.com/tektoncd/operator/pkg/reconciler/common.TargetVersion(0x2043728, 0xc001146820, 0xc00034b020, 0x3)
	github.com/tektoncd/operator/pkg/reconciler/common/releases.go:46 +0x9a
github.com/tektoncd/operator/pkg/reconciler/common.TargetManifest(0x2043728, 0xc001146820, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1d9d701, ...)
	github.com/tektoncd/operator/pkg/reconciler/common/releases.go:51 +0x74
github.com/tektoncd/operator/pkg/reconciler/common.AppendTarget(0x2019298, 0xc001178d80, 0xc001172dc0, 0x2043728, 0xc001146820, 0x1d15080, 0x1)
	github.com/tektoncd/operator/pkg/reconciler/common/stages.go:51 +0x77
github.com/tektoncd/operator/pkg/reconciler/common.Stages.Execute(0xc0009e7698, 0x4, 0x4, 0x2019298, 0xc001178d80, 0xc001172dc0, 0x2043728, 0xc001146820, 0x0, 0x0)
	github.com/tektoncd/operator/pkg/reconciler/common/stages.go:36 +0x82
github.com/tektoncd/operator/pkg/reconciler/kubernetes/tektondashboard.(*Reconciler).ReconcileKind(0xc000832e80, 0x2019298, 0xc001178d80, 0xc001146820, 0x0, 0x0)
	github.com/tektoncd/operator/pkg/reconciler/kubernetes/tektondashboard/tektondashboard.go:133 +0xe18
github.com/tektoncd/operator/pkg/client/injection/reconciler/operator/v1alpha1/tektondashboard.(*reconcilerImpl).Reconcile(0xc0001d3680, 0x2019298, 0xc001178d80, 0xc0011435f0, 0x9, 0xc001170358, 0x2019298)
	github.com/tektoncd/operator/pkg/client/injection/reconciler/operator/v1alpha1/tektondashboard/reconciler.go:249 +0x1352
knative.dev/pkg/controller.(*Impl).processNextWorkItem(0xc000394ae0, 0x0)
	knative.dev/pkg@v0.0.0-20210827184538-2bd91f75571c/controller/controller.go:555 +0x5e4
knative.dev/pkg/controller.(*Impl).RunContext.func3(0xc000e96000, 0xc000394ae0)
	knative.dev/pkg@v0.0.0-20210827184538-2bd91f75571c/controller/controller.go:491 +0x53
created by knative.dev/pkg/controller.(*Impl).RunContext
	knative.dev/pkg@v0.0.0-20210827184538-2bd91f75571c/controller/controller.go:489 +0x1a5
```

--- a/tekton/task-fetch-components.yaml
+++ b/tekton/task-fetch-components.yaml
@@ -48,6 +48,16 @@ spec:
           dir="trigger"
         fi
 
+        if [[ $comp == "dashboard" ]]; then
+          if [[ ${releaseFileName} == "tekton-dashboard-release" ]]; then
+            dir="dashboard/tekton-dashboard-fullaccess"
+          fi
+
+          if [[ ${releaseFileName} == "tekton-dashboard-release-readonly" ]]; then
+            dir="dashboard/tekton-dashboard-readonly"
+          fi
+        fi
+
         #nightly -> 0.0.0-nightly
         #latest -> find version till then -> 0.0.0-latest
         #version -> directory with version
@@ -73,7 +83,7 @@ spec:
 
         # before adding releases, remove existing version directories
         # ignore while adding for interceptors
-        if [[ ${releaseFileName} != "interceptors.notags" ]] ; then
+        if [[ ${releaseFileName} != "interceptors" ]] ; then
           rm -rf ${comp_dir}/*
         fi
 
@@ -96,6 +106,7 @@ spec:
           cp -r ${ko_data}/openshift/00-prereconcile ${comp_dir}/
           cp ${ko_data}/openshift/pipelines-rbac/* ${dirPath}/
         fi
+
 
         if [[ ${comp} == "dashboard" ]]; then
           sed -i '/aggregationRule/,+3d' ${dest}

